staging/src/k8s.io/kubectl/pkg/cmd/util/editor/editor.go

```diff
@@ -162,27 +161,7 @@ func (e Editor) LaunchTempFile(prefix, suffix string, r io.Reader) ([]byte, stri
 }
 
 func tempFile(prefix, suffix string) (f *os.File, err error) {
```

Comment:
Looks like this wrapping function isn't used elsewhere so let's remove it for extra cleanup!

Reply:
Definitely! Done